llap-tez/src/java/org/apache/hadoop/hive/llap/tezplugins/metrics/LlapTaskSchedulerMetrics.java

```diff
@@ -55,6 +61,9 @@
   private final JvmMetrics jvmMetrics;
   private final String sessionId;
   private final MetricsRegistry registry;
+  private final int decayMetricSampleSize;
+  private final double decayMetricAlphaFactor;
+  private Map<String, DaemonLatencyMetric> daemonTaskLatency = new ConcurrentHashMap<>();
```

Comment:
a comment, telling what the key is, would help.

Reply:
Done